application/backend/app/api/endpoints/sinks.py

```diff
@@ -97,7 +102,7 @@ async def create_sink(
 
     try:
         return configuration_service.create_sink(sink_config)
-    except ResourceAlreadyExistsError as e:
+    except ResourceWithNameAlreadyExistsError as e:
```

Comment:
We should handle the case where ids are provided as well, since all resources allow passing an ID during creation.